build/travis.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Fail on error
+set -e
+
+REL=$(dirname "$0")
+
+# Add everything, get ready for commit. But only do it if we're on
+# master. If you want to deploy on different branches, you can change
+# this.
+if [[ "$BRANCH" =~ ^master$|^[0-9]+\.[0-9]+\.X$ ]]; then
+    echo "Branch is master, so push new application to Quay registry"
+    export PATH=$HOME/bin:$PATH
+    curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.12.0/operator-sdk-v0.12.0-x86_64-linux-gnu -o $HOME/bin/operator-sdk-v0.12.0
+    chmod +x $HOME/bin/operator-sdk-v0.12.0
+    operator-courier verify --ui_validate_io deploy/olm-catalog/service-telemetry-operator
```

Comment:
This command and the one below it may(?) require `${REL}/../` in the paths in order to execute from anywhere outside the project root. I introduced this concept in this repo, but I'm not too strict on requiring that EVERYTHING work that way as long as we have some sort of consistent rule for which do and which don't.

Reply:
Since these are run from the location relative to the `.travis.yml` it's not necessary since it was working prior to these changes :)

I could probably move these installs to separate `build/travis/[...].sh` scripts but that's starting to seem like overkill for this particular patch set. What do you think?

Reply:
I think that's overkill, yeah. I only commented because you used REL inconsistently (on some lines but not others)